### PR TITLE
cUSD Liquidity CGP 0061

### PR DESCRIPTION
**Pull request by rcroessmann**

### CODE REVIEW

**Review by willkraft — APPROVED** [review]

File: CGPs/cgp-0061/cpg-0061.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    "args": [
+      "0x18e6bfdc909063f7445e410a5495264619495bcb"
+    ],
+    "value": "0"
+  },
+  {
+    "contract": "StableToken",
+    "function": "approve",
+    "args": [
+      "TODO:MOBIUS_POOL_ADDRESS",
```
Comment (rcroessmann) [inline CGPs/cgp-0061/cpg-0061.json:31]:
```suggestion
      "0xFa3df877F98ac5ecd87456a7AcCaa948462412f0",
```

File: CGPs/cgp-0061/cpg-0061.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      "5000000000000000000000000"
+    ],
+    "value": "0"
+  },
+  {
+    "contract": "0x37f750B7cC259A2f741AF45294f6a16572CF5cAd",
+    "function": "approve(address,uint256)",
+    "args": [
+      "TODO:MOBIUS_POOL_ADDRESS",
```
Comment (rcroessmann) [inline CGPs/cgp-0061/cpg-0061.json:40]:
```suggestion
      "0xFa3df877F98ac5ecd87456a7AcCaa948462412f0",
```

File: CGPs/cgp-0061/cpg-0061.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      "5000000000000"
+    ],
+    "value": "0"
+  },
+  {
+    "contract": "TODO:MOBIUS_POOL_ADDRESS",
```
Comment (rcroessmann) [inline CGPs/cgp-0061/cpg-0061.json:46]:
```suggestion
    "contract": "0xFa3df877F98ac5ecd87456a7AcCaa948462412f0",
```

File: CGPs/cgp-0061/cpg-0061.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    ],
+    "value": "0"
+  },
+  {
+    "contract": "0xFa3df877F98ac5ecd87456a7AcCaa948462412f0",
+    "function": "addLiquidity(uint256[],uint256,uint256)",
+    "args": [
+      [
+        "5000000000000000000000000",
+        "5000000000000"
```
Comment (rcroessmann) [inline CGPs/cgp-0061/cpg-0061.json:51]: USDC has only 6 decimal points

File: CGPs/cgp-0061/cpg-0061.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      ],
+      0,
+      1661419596
```
Comment (rcroessmann) [inline CGPs/cgp-0061/cpg-0061.json:54]: Deadline parameter set to August 25th

File: CGPs/cgp-0061.md [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+* USDC (Wormhole)
+* LPTokens from the StableSwap pool.
+
+Our aim is for Governance to hold these tokens for most steps throughout the process, but due to limitations in the setup, a 4/6 Multisig (GnosisSafe) on Ethereum will be used as an intermediary. It will have signers from Mento and cLabs:
+
+* @bowd (Mento)
+* @Nadiem (Mento)
+* @roman (Mento)
+* @nraghuveera (cLabs) 
+* @tim (cLabs)
```
Comment (rcroessmann) [inline CGPs/cgp-0061.md:96]:
```suggestion
* @nraghuveera (cLabs)
```

File: CGPs/cgp-0061.md [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+* USDC (Wormhole)
+* LPTokens from the StableSwap pool.
+
+Our aim is for Governance to hold these tokens for most steps throughout the process, but due to limitations in the setup, a 4/6 Multisig (GnosisSafe) on Ethereum will be used as an intermediary. It will have signers from Mento and cLabs:
+
+* @bowd (Mento)
+* @Nadiem (Mento)
+* @roman (Mento)
+* @nraghuveera (cLabs) 
+* @nraghuveera (cLabs)
```
Comment (rcroessmann) [inline CGPs/cgp-0061.md:96]:
```suggestion
* @nimittank (cLabs)
```

**Review by calicokittencat — APPROVED** [review]

**Pull request merged**